tests/testing.py

```diff
@@ -39,4 +39,4 @@ def _file_contents(self, filename):
             return file.read()
 
     def _file_contents_eq(self, filename, contents):
-        eq_(self._file_contents(filename), contents)
+        assert self._file_contents(filename) == contents
```

Comment:
Unfortunately, pytest doesn't show the strings when this fails; it just says AssertionError.

Reply:
It does if you run it in verbose mode see #104

Reply:
Hmm, not for me:

```
% pytest -vv -k test_autoclass_members_list
======================================= test session starts ========================================
platform darwin -- Python 3.7.3, pytest-5.1.2, py-1.8.0, pluggy-0.12.0 -- /usr/local/opt/python/bin/python3.7
cachedir: .pytest_cache
rootdir: /Users/erose/Checkouts/sphinx-js
collected 45 items / 43 deselected / 2 selected

tests/test_build/test_build.py::Tests::test_autoclass_members_list FAILED                    [ 50%]
tests/test_build/test_build.py::Tests::test_autoclass_members_list_star FAILED               [100%]

============================================= FAILURES =============================================
________________________________ Tests.test_autoclass_members_list _________________________________

self = <test_build.Tests testMethod=test_autoclass_members_list>

    def test_autoclass_members_list(self):
        """Make sure including a list of names after ``members`` limits it to
        those names and follows the order you specify."""
        self._file_contents_eq(
            'autoclass_members_list',
>           'class ClosedClass()\n\n   Closed class.\n\n   ClosedClass.publical3()\n\n      Public thing 3.\n\n   ClosedClass.publical()\n\n      Public thing.\n')

tests/test_build/test_build.py:152:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <test_build.Tests testMethod=test_autoclass_members_list>
filename = 'autoclass_members_list'
contents = 'class ClosedClass()\n\n   Closed class.\n\n   ClosedClass.publical3()\n\n      Public thing 3.\n\n   ClosedClass.publical()\n\n      Public thing.\n'

    def _file_contents_eq(self, filename, contents):
>       assert self._file_contents(filename) == contents
E       AssertionError

tests/testing.py:42: AssertionError
```

Reply:
I think it's that the comparator routine is in a non-test helper module, so its assertions aren't getting rewritten. Hmm, there are disadvantages to fancy static analysis. :-( Trying to get it to work by calling register_assert_rewrite('sphinx_js.tests.testing'), but no luck yet.

Reply:
There. Got it going. It needed `register_assert_rewrite('tests.testing')`.